semperfiwebdesign/all-in-one-seo-pack
# Improvements to HTML / CSS for markup validation

Issue #2534

## Proposed changes
This introduces numerous improvements to our HTML and CSS markup in order to pass the W3C validation.  

## Types of changes
- Improves existing code

## Checklist

## Testing instructions
Test all screens in all browsers, paying specific attention to the following:

 - Width of the settings screens when viewed in various browser widths
 - The plugin title in the About box on the settings screens
 - The link to Github in the About box on the settings screens
 - The bullet list in the About box on the settings screens
 - The contents of the Support box on the settings screens
 - The links in the Support box on the settings screens
 - Closing and opening the meta boxes on the settings screens
 - Saving settings on the settings screens
 - Layout of the options on the settings screens
 - The Additional Pages table on the Sitemap module
 - The table on the Robots module
 - The fields with character counters in the Home Page Settings section on the Open Graph module
 - The OG image on the Add New Post / Edit Post screen
 - The Debug This Post button on the Add New Post / Edit Post screen

Verify all CSS and JavaScript is loading correctly.  Check the code changes and let me know if you have any questions.
